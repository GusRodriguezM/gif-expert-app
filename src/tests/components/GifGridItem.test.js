import React from 'react';
import '@testing-library/jest-dom';
import {shallow} from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('Tests in <GrifGridItem/>', () => {
   
    let wrapper = shallow(<GifGridItem/>);

    
    test('Should show <GrifGridItem/> correctly', () => {

        expect(wrapper).toMatchSnapshot();

    });
    
});

