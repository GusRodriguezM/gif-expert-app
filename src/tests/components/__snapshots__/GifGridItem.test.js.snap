// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <GrifGridItem/> Should show <GrifGridItem/> correctly 1`] = `
<div
  className="card animate__animated animate__fadeInLeftBig animate__delay-2s"
>
  <img />
  <p />
</div>
`;
